social/tasks.py

```diff
@@ -0,0 +1,44 @@
+import feedparser
+import re
+
+from background_task import background
+
+from social.app.models.author import Author
+from social.app.models.post import Post
+
+# Get the GitHub activity of a user
+# Reference source: https://pypi.python.org/pypi/django-background-tasks
+@background(schedule=60)
+def get_github_activity(authorId):
+    gitAuthor = Author.objects.get(id=authorId)
```

Comment:
Minor stylistic feedback but Python PEP-8 prefers use of underscores over Java's camelCasing.